rts/lra-examples/cdi-participant/pom.xml

```diff
@@ -101,19 +111,27 @@
             <version>${version.microprofile.lra}</version>
         </dependency>
 
+       <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5</artifactId>
```

Comment:
The rest-assured and junit is probably not needed.

Reply:
+1